packages/ui/components/EditorSwitch.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useCallback } from 'react'
```

Comment:
The component is not usable with the keyboard. My recommendation is that you wrap it with https://www.radix-ui.com/docs/primitives/components/switch.